addons/lunch/models/lunch.py

```diff
@@ -0,0 +1,296 @@
+# -*- coding: utf-8 -*-
+
+import datetime
+from openerp import models, fields, api, _
+
+class lunch_order(models.Model):
+    """
+    lunch order (contains one or more lunch order line(s))
+    """
+    _name = 'lunch.order'
+    _description = 'Lunch Order'
+    _order = 'date desc'
+
+    @api.multi
+    def name_get(self):
+        return [(order.id, _("Lunch Order #%d") % order.id) for order in self]
```

Comment:
I suggest to use '%s %s' % (_('Lunch Order'), '#%d' % order.id) as string. This way people won't have to translate #%d in their specific language, only Lunch Order, which makes more sense.